internal/settings/settings.go

```diff
@@ -29,6 +31,23 @@ func (o *Options) Validate() error {
 	if len(o.ModulePaths) != 0 && len(o.ExcludeModulePaths) != 0 {
 		return fmt.Errorf("at most one of `rootModulePaths` and `excludeModulePaths` could be set")
 	}
+
+	// TODO: abstract out this validation logic
+	// since it's being used in both the places
```

Comment:
@radeksimko does it make sense to abstract out this validation for paths to a separate package? something like `fileutil` or maybe rename the existing `pathcmp` package to path and have such utilities moved there?